apps/L1G/restaurant/restaurant-frontend/src/components/home/HomePageContainer.tsx

```
@@ -0,0 +1,56 @@
+'use client';
+import { Button } from '@/components/ui/button';
+import { useState } from 'react';
+import MenuCard from './MenuCard';
+import { useGetCategoriesQuery, useGetFoodsQuery } from '@/generated';
+
+const HomePageContainer = () => {
+  const { data: foodsData, loading: foodsLoading } = useGetFoodsQuery();
```

ESLint

[error] 'foodsLoading' is assigned a value but never used.

(no-unused-vars)